src/participant/prompts/namespace.ts

```diff
@@ -27,6 +28,14 @@ COLLECTION_NAME: sightings
 ___
 Example 2:
 
+User: How do I create an index in my pineapples collection?
+
+Response:
+COLLECTION_NAME: pineapples
```

Comment:
Should we import `DB_NAME_ID` and `COL_NAME_ID` and use them in the prompt rather than hardcoding the values? Not critical, but in case we ever want to tweak them, we'll have to update several places instead of one.